bin/rl/rl.cpp

```diff
@@ -4167,8 +4170,8 @@ ProcessConfig
 
       // Walk the condition nodes looking for applicability.
 
-      ConditionNodeList * conditionNodeList = NULL;
-      ConditionNodeList * conditionNodeLast = NULL;
+      conditionNodeList = NULL;
+      conditionNodeLast = NULL;
```

Comment:
Hoisting these declarations above the loop is not useful. They are already scoped to the loop. Additionally, this change is redundant as pertains to the assignment of NULL at initialization time. What problem is this trying to solve that the compiler cannot figure out (e.g. hoisting of declarations)?